data-service-model/src/main/java/com/amazonaws/blox/dataservicemodel/v1/model/operations/DescribeEnvironmentRevisionResponse.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservicemodel.v1.model.operations;
+
+import com.amazonaws.blox.dataservicemodel.v1.model.EnvironmentRevision;
+import lombok.Value;
+
+@Value
+public class DescribeEnvironmentRevisionResponse {
+  private final EnvironmentRevision environmentRevision;
```

Comment:
I don't think we should return an object here so we can change the fields of different response objects independently. So if we need to change describEnvResponse we don't have to change createEnvResponse (ran into this problem before)

Reply:
Could you give an example where that would apply to EnvironmentRevisions specifically? I can sort of see the use case for entities that are mutable, and can have some properties unset at creation time, but since EnviornmentRevisions are immutable, it seems like we couldn't have that situation.

Reply:
It was more of a general comment for all response types but for this one, I'm not sure, what if we add some timestamp or metadata later?

Reply:
Then those would be sibling members of the `environmentRevision` property on the response object? The only thing I can think of is something that shows e.g. environment status, but that would be an `EnvironmentRevisionStatus` object then, not a revision.

Reply:
yes siblings